contrib/renum/src/Makefile.am

```diff
@@ -1,4 +1,4 @@
-bin_PROGRAMS = lepton-renum
+noinst_PROGRAMS = lepton-renum
```

Comment:
I cannot remember/understand why it should be so. On my system, I cannot install things from `contrib/` even if I specify `--enable-contrib` on the configure stage.

Reply:
I can't remember, too.
It seems `contrib/` tools were never meant to be installed:
look at the git history of `Makefile.am`'s.
`noinst_` has always been there.

Reply:
OK, I see.  That pretty clear means the `--enable-contrib` option is useless and the `noinst_` prefix is abused since it was designed for supplementary tools that help to build other tools, if I understood correctly from autotools' documentation.

Reply:
And, apropos, I recall why I agreed with that status of tools in `contrib/`.  Because their names still conflict with those in the `geda-gaf` package.